src/kOS.Safe/Compilation/Opcode.cs

```diff
@@ -1680,41 +1680,18 @@ public override string ToString()
     
     public class OpcodeAddTrigger : Opcode
     {
-        [MLField(1,false)]
-        private bool ShouldWait { get; set; }
-
```

Comment:
Again, it feels good to fix something by getting to *cut* code and replace it with .. nothing.

The ```ShouldWait``` parameter to the ```addtrigger``` opcode only served one purpose and that was to differentiate the triggers created by WAIT from the ones created by other kinds of trigger statements like WHEN and ON.  It's gone now because there should be no such thing as a "waiting" trigger like this anymore.